pkg/helm/index.go

```diff
@@ -0,0 +1,48 @@
+package helm

```

Comment:
this is the dep package so you should not change it?

Reply:
This is `pkg` not `vendor`. Therefore, it is not a dep package.

Reply:
ah...ok :)